packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -95,13 +96,13 @@ export const listHeader = mergeStyles({
  * @private
  */
 export const participantListStack = mergeStyles({
-  width: '15rem',
   height: '100%'
 });
 
 /**
  * @private
  */
 export const participantListStyle = mergeStyles({
-  overflowY: 'hidden'
+  height: '100%',
+  overflow: 'auto'
```

Comment:
Do you know why we had 'hidden' earlier? Is it the case the the implementation of ParticipantList provides its own way to scroll? Or was this just a bug?

Reply:
bug. The List of participant in chat was not working properly (no vertical scrollbar when there were a lot of people in the chat, so list truncated before)